horovod/common/operations.cc

```diff
@@ -0,0 +1,1577 @@
+// Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
How is this copyright tensorflow? did this file start from there? We should have Uber copyright?

Reply:
We do have Uber copyright there as well.